tstest/integration/integration_test.go

```diff
@@ -312,6 +314,33 @@ func TestOneNodeUpAuth(t *testing.T) {
 	d1.MustCleanShutdown(t)
 }
 
+func TestConfigFileAuthKey(t *testing.T) {
+	tstest.SkipOnUnshardedCI(t)
```

Comment:
@maisem, FYI I had to add this. Not because this test fails on CI, but because this test is the straw that breaks the GitHub Actions back timing wise. You can actually do this to any test in this whole package to make it pass.

But the unsharded builders that run this package are all stupid slow in general (taking about 10 minutes?!?) and this pushes it over.